TraditionalBridge.UnitTest/Dispatcher.dbl

```diff
@@ -5,30 +5,308 @@ import System.Text.Json
-			DispatchRunner.EnsureSuccess(dispatchResult)
-		endmethod
-	endclass
+            endparams
+            record
+                dispatchResult, @string
+        proc
+            dispatchResult = DispatchRunner.RemoveMessageHeader(DispatchRunner.RunDispatch('{"jsonrpc":"2.0","id":3,"method":"arbitrario_optional","params":[{"ReturnedValue":true,"DataType":16},{"PassedValue":9,"DataType":8},{"PassedValue":"World","DataType":1},{"PassedValue":"","DataType":1},{"DataType":8}]}'))
+            DispatchRunner.EnsureSuccess(dispatchResult)
+        endmethod
+.endregion
+
+.region "Get"
+        {TestMethod}
+        public method GetAllCustomers, void
+            endparams
+            record
+                dispatchResult, @string
+        proc
+            dispatchResult = DispatchRunner.RemoveMessageHeader(DispatchRunner.RunDispatch('{"jsonrpc":"2.0","id":4,"method":"GetAllCustomers", "params":[{"ReturnedValue":true,"DataType":64}]}'))
+            DispatchRunner.EnsureSuccess(dispatchResult)
+        endmethod
+
+        {TestMethod}
+        public method GetCustomerArray, void
+            endparams
+            record
+                dispatchResult, @string
+        proc
+            dispatchResult = DispatchRunner.RemoveMessageHeader(DispatchRunner.RunDispatch('{"jsonrpc":"2.0","id":5,"method":"GetCustomerArray", "params":[{"ReturnedValue":true,"DataType":64}]}'))
+            DispatchRunner.EnsureSuccess(dispatchResult)
+        endmethod
+
+        {TestMethod}
+        public method GetAllItems, void
+            endparams
+            record
+                dispatchResult, @string
+        proc
+            dispatchResult = DispatchRunner.RemoveMessageHeader(DispatchRunner.RunDispatch('{"jsonrpc":"2.0","id":6,"method":"GetAllItems", "params":[{"ReturnedValue":true,"DataType":64}]}'))
+            DispatchRunner.EnsureSuccess(dispatchResult)
+        endmethod
+
+        {TestMethod}
+        public method GetAllOrders, void
+            endparams
+            record
+                dispatchResult, @string
+        proc
+            dispatchResult = DispatchRunner.RemoveMessageHeader(DispatchRunner.RunDispatch('{"jsonrpc":"2.0","id":7,"method":"GetAllOrders", "params":[{"ReturnedValue":true,"DataType":64}]}'))
+            DispatchRunner.EnsureSuccess(dispatchResult)
+        endmethod
+
+        {TestMethod}
+        public method GetAllVendors, void
+            endparams
+            record
+                dispatchResult, @string
+        proc
+            dispatchResult = DispatchRunner.RemoveMessageHeader(DispatchRunner.RunDispatch('{"jsonrpc":"2.0","id":8,"method":"GetAllVendors", "params":[{"ReturnedValue":true,"DataType":64}]}'))
+            DispatchRunner.EnsureSuccess(dispatchResult)
+        endmethod
+.endregion
+
+.region "Parameters"
+; Commented out because no support for HANDLE
```

Comment:
Is the lack of support for HANDLE on the HC side or the language/tooling side? If HC side, I would rather see an `{Ignore("Missing support for HANDLE")}` instead of commenting it out.

Reply:
used ignore now